addons/account/models/account.py

```diff
@@ -1067,18 +1067,20 @@ def _default_tax_group(self):
 
     @api.model
     def default_get(self, vals):
-        rslt = super(AccountTax, self).default_get(vals)
+        # company_id is added so that we are sure to fetch a default value from it to use in repartition lines, below
+        rslt = super(AccountTax, self).default_get(vals + ['company_id'])
```

Comment:
I don't see what this is doing in this branch.

Reply:
@jco-odoo Many bug in master it's already solved in saas-12.3 so we temporary cherry pick in this branch. 
because a commit is not carry forward in master at this time.
A tester can continue its work it's the main reason behind this.